Documentation/API JSON.json

```
@@ -3,18 +3,118 @@
   "info": {
     "title": "HotelBookingAPI",
     "description": "API for managing hotel bookings and room availability",
-    "version": "1.0.0"
+    "version": "v1"
```

Können wir an der Stelle das "1.0.0" lassen, damit das semantic versioning bestehen bleibt?
"/v1/" ist dann weiter unten eh Teil der URL
```suggestion
    "version": "1.0.0"
```